cmd/minishift/cmd/config/config.go

```diff
@@ -100,6 +100,14 @@ var settings []Setting = []Setting{
 		callbacks:   []setFn{RequiresRestartMsg},
 	},
 	{
+		name: "hostfolders-automount",
```

Comment:
I stand to my point as well :-) I am looking at where we are coming from there and that is the behaviour of docker-machine and ADB/CDK. Here we have the users home directory mounted per default (either via guest additions or vagrant-sshfs, or whatever). This is to a certain degree what we try to mimic here. I just want my home directory mounted in the VM, preferably w/o any interaction. 

I understand that with CIFS we needs this additional configuration step and that's fine, but having to do it eat single time I create an instance? 

I really like that host folders become more explicit now and that I can have multiple folders mounted. This is much better than the plain "we just mount your home directory". It opens up new ways of using host folders. However, there is big loss in convenience, if my config disappears every 'minishift delete'.

> To me, an instance can represent a set of work and services that might have different storage needs.

I hear where you are coming from, but I don't think this is the most common use case. You are envisioning something where I have one instance per app (project) and I mount for example just dedicated directories, specific to this app (opposed to the whole user directory). Is this how you are thinking?

> But when more than one instance is considered, why would I want to mount my homefolder in all of them?

Because in most cases your home folder will contain somewhere the workspace you want to share. And if it is outside your home directory, then you can define another host folder. That's the big win of what we have now.

> It might be me... but I do understand the convenience aspect. I just wonder how often this will be the case.

I think this is the default case. This is also the case we have been "selling" so far. "Look your home directory is in the VM". As a developer, I'd rather know that I have my home directory mounted at all times opposed to having to work out multiple finder grained host folder mounts